src_c/math.c

```diff
@@ -756,7 +758,8 @@ vector_inplace_floor_div(pgVector *o1, PyObject *o2)
 static PyObject *
 vector_neg(pgVector *self)
 {
-    pgVector *ret = (pgVector *)pgVector_NEW(self->dim);
+    pgVector *ret = (pgVector *)_vector_subtype_new(Py_TYPE(self), self->dim);
+
     if (ret != NULL) {
         Py_ssize_t i;
```

Comment:
This isn't your fault, but I noticed almost all the rest check if the value is NULL and return NULL there. You could change this method to be consistent with that.